bindings/python/setup.py

```diff
@@ -19,7 +19,7 @@
     description="Python bindings for libmongocrypt",
     long_description=LONG_DESCRIPTION,
     packages=find_packages(),
-    install_requires=["cffi>=1.12.0,<2"],
+    install_requires=["cffi>=1.12.0,<2", "cryptography>=2.7,<3"],
```

Comment:
cryptography 2.7 is the latest release. I haven't attempted to use earlier releases yet but we may be able to do `"cryptography>=2,<3"` instead. That would provide greater compatibility for users who already have cryptography installed.

Reply:
Do some spot testing, back to the earliest 2.x version to ensure they provide what we need.

Reply:
I'll do this in https://jira.mongodb.org/browse/PYTHON-1953.